src/components/separators/_horizontal-separator.scss

```diff
@@ -20,6 +20,10 @@ $includeHtml: false !default;
         margin: (rhythm(1) - $horizontalSeparatorHeight) 0 rhythm(1);
       }
     }
+
+    &--small-spaced {
```

Comment:
https://github.com/brainly/style-guide/blob/master/CONTRIBUTING.md#modifier-dictionary

Hm maybe it shout be `short` then WDYT ?